_topic_map.yml

```diff
@@ -773,8 +773,8 @@ Topics:
   File: installing-pipelines
 - Name: Uninstalling OpenShift Pipelines
   File: uninstalling-pipelines
-- Name: Creating applications with OpenShift Pipelines
-  File: creating-applications-with-cicd-pipelines
```

Comment:
This just came up yesterday - because our redirects list is getting massive, Vikram asked that we avoid changing the files as much as possible and just change the name.

Reply:
Ack, fixed, and reverted all changes wrt to this change.